packages/server/src/config.ts

```diff
@@ -16,7 +16,12 @@ const config = {
     password: env.CENTRIFUGE_ADMIN_PASSWORD || 'admin',
     // Centrifuge Identity Address
     account: env.CENTRIFUGE_ADMIN_ACCOUNT || '0xBeaB9D94D23Ff609b92b8C739f4EE13bCe918F8c',
-    permissions: [PERMISSIONS.CAN_MANAGE_USERS, PERMISSIONS.CAN_MANAGE_SCHEMAS, PERMISSIONS.CAN_MANAGE_DOCUMENTS],
+    chainAccount: {centrifugeChainAccount : {
```

Comment:
should this be hardcoded? Or passed through env?

Reply:
i can pass it through the env 👍